addons/mail/static/src/new/core/chat_window_model.js

```diff
@@ -66,7 +66,7 @@ export class ChatWindow {
      */
     constructor(state, data) {
         Object.assign(this, {
-            threadLocalId: data.threadLocalId,
+            thread: data.thread,
```

Comment:
I'm not sure to understand on this model thread is both assigned as field and has a getter?

Reply:
Yes, I defined `set thread`, so it invokes it here, which does `threadLocalId = thread.localId`.